java/runtime/src/test/java/io/ray/runtime/functionmanager/FunctionManagerTest.java

```diff
@@ -117,6 +130,32 @@ public void testGetFunctionFromFunctionDescriptor() {
     });
   }
 
+  @Test
+  public void testInheritance() {
+    final FunctionManager functionManager = new FunctionManager(null);
+    fooDescriptor = new JavaFunctionDescriptor(Base.class.getName(), "foo",
+        "()Ljava/lang/Object;");
+    Assert.assertEquals(functionManager.getFunction(JobId.NIL, fooDescriptor).functionDescriptor,
+        fooDescriptor);
+    Assert.assertEquals(functionManager.getFunction(JobId.NIL, fooDescriptor)
+            .executable.getDeclaringClass(), Base.class);
+    RayFunction fooFunc = functionManager.getFunction(JobId.NIL,
+        new JavaFunctionDescriptor(Bar.class.getName(), "foo",
+            "()Ljava/lang/Object;"));
+    Assert.assertEquals(fooFunc.executable.getDeclaringClass(), Base.class);
```

Comment:
Could you add some comments in this test case? e.g. the reason we need this assertion.